crates/qt-build-utils/src/lib.rs

```diff
@@ -531,7 +531,8 @@ impl QtBuild {
     /// as well as the path to the generated metatypes.json file, which can be passed to [register_qml_module](Self::register_qml_module).
     ///
     /// * uri - Should be passed if the input_file is part of a QML module
-    pub fn moc(&mut self, input_file: impl AsRef<Path>, uri: Option<&str>) -> MocProducts {
+    /// * include_paths - Additional include paths to pass to moc
+    pub fn moc(&mut self, input_file: impl AsRef<Path>, uri: Option<&str>, include_paths: &Vec<PathBuf>) -> MocProducts {
```

Comment:
we should go through the other `moc` arguments while we are here and consider which others might be useful.